assembly/__tests__/plaindate.spec.ts

```diff
@@ -1112,13 +1114,13 @@ describe("Date.equal works", () => {
 //   it('>=', () => throws(() => d1 >= d2));
 // });
 
-// describe('Min/max range', () => {
-//   it('constructing from numbers', () => {
-//     throws(() => new PlainDate(-271821, 4, 18), RangeError);
-//     throws(() => new PlainDate(275760, 9, 14), RangeError);
-//     expect(new PlainDate(-271821, 4, 19).toString()).toBe('-271821-04-19');
-//     expect(new PlainDate(275760, 9, 13).toString()).toBe('+275760-09-13');
-//   });
+describe('Min/max range', () => {
+  it('constructing from numbers', () => {
+    // throws("RangeError", () => { new PlainDate(-271821, 4, 18) });
+    // throws("RangeError", () => { new PlainDate(275760, 9, 14) });

```

Comment:
this still not work

Reply:
I presume you could use this? https://github.com/ColinEberhardt/assemblyscript-temporal/blob/master/assembly/date.ts#L155

Reply:
Unfortunately no. `epochFromParts` it seems required `u128` arithmetic (bignum sum)

Reply:
we could use as-bignum but probably we could avoid this. Need to think